tests/unit/helpers.js

```diff
@@ -787,6 +787,23 @@ export function getFakeAddonManagerWrapper({
   };
 }
 
+export function getFakeAddonManagerWrapperWithJest({
```

Comment:
There wouldn't yeah. Since I wasn't 100% done I haven't removed `test_installAddon.js` yet, once I do I'll replace `getFakeAddonManagerWrapper` entirely as you suggest.